src/cultionet/data/create.py

```diff
@@ -274,7 +275,7 @@ def create_image_vars(
     gain: float = 1e-4,
     offset: float = 0.0,
     grid_edges: T.Optional[gpd.GeoDataFrame] = None,
-    ref_res: T.Optional[float] = 10.0,
+    ref_res: T.Optional[T.Union[float, T.Tuple[float, float]]] = 10.0,
```

Comment:
so here you also accept tuples of values?

Reply:
@nnguyen622 the default cell size is now `None` (see the commit in `args.yml`). This change allows the resolution to be gathered from the image (e.g., in case the user doesn't know the exact cell size). If taken from the image, and in cases of non-square pixels, tuples are allowed.